cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -274,6 +276,187 @@ sleep INF
 		})
 	})
 
+	Context("Rootless vhostnet", func() {
+		var sriovDevice *sriovv1.InterfaceExt
```

Comment:
minor:
In most of cases across the project we use single 
var (
....
)
in order to define multiple variables.

Reply:
for global vars yes, but not for local ones